lib/modules.js

```diff
@@ -131,8 +136,13 @@ export async function updateSourceFiles ({ module, ipnsKey, moduleVersionsDir })
   const cid = await getLatestCID(ipnsKey)
   const isUpdate = lastSeenCID !== cid
   if (!isUpdate) {
-    console.error(`[${module}]  ✓ no update available`)
-    return isUpdate
+    try {
+      await stat(outDir)
```

Comment:
This is another precaution: Station Desktop was also failing because according to the `latest` file we had an installation of the module, but the files weren't on disk.